### ISSUE TITLE
Upgrade react-scripts and other packages

### ISSUE THREAD

**Issue by gianpaj** [helpmap/helpmap.io#20]
```
yarn outdated
yarn outdated v1.22.5
warning ../package.json: No license field
info Color legend :
 "<red>"    : Major Update backward-incompatible updates
 "<yellow>" : Minor Update backward-compatible features
 "<green>"  : Patch Update backward-compatible bug fixes
Package                     Current      Wanted  Latest Package Type    URL
@appbaseio/reactivesearch   3.7.1        3.12.4  3.12.4 dependencies    https://github.com/appbaseio/reactivesearch#readme
@material-ui/core           3.9.3        3.9.4   4.11.0 dependencies    https://material-ui.com/
@material-ui/icons          3.0.2        3.0.2   4.9.1  dependencies    https://github.com/mui-org/material-ui/tree/master/packages/material-ui-icons
antd                        3.16.1       3.26.18 4.6.5  dependencies    https://ant.design
appbase-js                  4.0.2-beta.5 4.1.1   4.1.1  dependencies
eslint-config-prettier      6.7.0        6.11.0  6.11.0 devDependencies https://github.com/prettier/eslint-config-prettier#readme
eslint-plugin-prettier      3.1.2        3.1.4   3.1.4  devDependencies https://github.com/prettier/eslint-plugin-prettier#readme
eslint-plugin-react         7.17.0       7.21.1  7.21.1 devDependencies https://github.com/yannickcr/eslint-plugin-react
node-sass                   4.13.1       4.14.1  4.14.1 dependencies    https://github.com/sass/node-sass
prettier                    1.19.1       1.19.1  2.1.2  devDependencies https://prettier.io
react-google-maps-loader    4.2.5        4.3.0   4.3.0  dependencies    https://github.com/cedricdelpoux/react-google-maps-loader#readme
react-google-places-suggest 3.7.1        3.9.2   3.9.2  dependencies    https://github.com/xuopled/react-google-places-suggest#readme
react-intl                  2.8.0        2.9.0   5.8.2  dependencies    https://formatjs.io/docs/react-intl
react-places-autocomplete   7.2.1        7.3.0   7.3.0  dependencies    https://github.com/kenny-hibino/react-places-autocomplete#readme
react-scripts               3.4.1        3.4.1   3.4.3  dependencies    https://github.com/facebook/create-react-app#readme
semantic-ui-react           0.86.0       0.86.0  1.3.0  dependencies    https://github.com/Semantic-Org/Semantic-UI-React#readme
✨  Done in 3.99s.
```

**Comment by rarchitgupta** [comment]
I'd like to take this up. Just to be clear, I've to update the packages to the versions in the "wanted" column right?

**Comment by gianpaj** [comment]
hey @rarchitgupta thanks for volunteering!

yes for the moment only minor upgrades. unless there's a good reason to upgrade to a major version and it's easy to upgrade. Of course, the website should continue to work as before :)

let me know if you have any questions

**Comment by gianpaj** [comment]
Fixed by #24
